lymproblems/timezonechange.py

```diff
@@ -0,0 +1,20 @@
+# Change the system's timezone to the same of SFO, USA
+
+import os
+import sys
+import pytz
+from datetime import datetime
+from .utils import system, success, fail
+
+
+def setup():
+    "Setup timezonechange"
+    pass  # Nothing to do.
+
+
+def verify():
+    "Verify timezonechange"
+
+    if os.popen("date +%T").read().strip("\n") != datetime.now(pytz.timezone("US/Pacific")).strftime("%H:%M:%S"):
```

Comment:
Verify it via system level, which link needs to be changed to `US/Pacific` for this to be working?

Remember, here we are verifying the status of the system.

Reply:
@kushaldas I made the changes as asked. Please review it again.